policyengine_us/parameters/gov/states/or/liheap/payout_region1.yaml

```
@@ -0,0 +1,189 @@
+description: Payout from LIHEAP for Region 1 counties in the state of Oregon
+
+2021-10-01:
+  spm_unit: 1
+    IncomeRange: 1:
+      Electricity_Heating: 410
+      HeatingOil: 600
+      LiquidGas: 600
+      NaturalGas: 435
```

We need to pick specific variables which each breakdown range will be based on. 

```suggestion
2021-10-01:
  1:
    1:
      Electricity_Heating: 410
      HeatingOil: 600
      LiquidGas: 600
      NaturalGas: 435
```

---

We will have the breakdown categories in the metadata